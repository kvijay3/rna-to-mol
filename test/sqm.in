Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6    C2        6.4190        8.2190       -3.2520 
   7    N1        7.2020        7.5410       -4.1200 
   6    C6        7.1540        6.1900       -4.1140 
   6    C5        6.3210        5.5560       -3.1670 
   6    C4        5.5630        6.3820       -2.3480 
   7    N3        5.5580        7.7270       -2.3360 
   7    N9        4.7890        5.5340       -1.5960 
   6    C8        5.1400        4.2540       -1.9440 
   7    N7        6.0720        4.2280       -2.8760 
   6   C1*        3.6560        5.9370       -0.7900 
   6   C2*        3.6770        5.6130        0.7270 
   6   C3*        2.2840        5.0090        1.0070 
   6   C4*        1.4960        5.2890       -0.2740 
   8   O4*        2.4780        5.3000       -1.3290 
   6   C5*        0.7300        6.6210       -0.3060 
   8   O5*        0.0460        6.7580       -1.5520 
   8   O3*        1.6060        5.4880        2.1580 
  15     P        2.1890        5.1370        3.6170 
   8   O1P        3.3870        4.2480        3.6380 
   8   O2P        2.4740        6.6030        4.1980 
   8   O5*        0.9150        4.6250        4.4750 
   6   C5*        1.1750        4.2890        5.8400 
   6   C4*       -0.0690        3.8560        6.6410 
   6   C3*       -0.7800        2.5960        6.1280 
   6   C2*       -2.2700        2.9460        6.1630 
   6   C1*       -2.3420        4.2980        6.8800 
   8   O4*       -1.0460        4.9130        6.7190 
   7    N9       -2.7220        4.3460        8.2820 
   6    C8       -3.4520        5.3830        8.8260 
   7    N7       -3.7180        5.2000       10.1010 
   6    C5       -3.1310        4.0010       10.3980 
   6    C4       -2.5120        3.4470        9.2950 
   7    N3       -1.8550        2.2590        9.2230 
   6    C2       -1.8130        1.6180       10.3680 
   7    N1       -2.3970        2.0780       11.5310 
   6    C6       -3.0880        3.2750       11.6490 
   8    O6       -3.5880        3.6330       12.7090 
   7    N2       -1.1750        0.4280       10.4510 
   8   O2*       -3.1400        1.9280        6.6380 
   8   O3*       -0.4600        1.4620        6.9400 
   8   O2*        3.9250        6.7540        1.5620 
   7    N6        7.9150        5.5000       -5.0430 
   1    H2        6.4910        9.3020       -3.3080 
   1    H8        4.6620        3.3820       -1.5190 
   1   H1*        3.5300        7.0160       -0.9360 
   1   H2*        4.4540        4.8750        0.9570 
   1   H3*        2.3960        3.9200        1.0930 
   1   H4*        0.7930        4.4750       -0.4850 
   1  H5*1        1.4000        7.4810       -0.2200 
   1  H5*2       -0.0130        6.6780        0.4920 
   1   H5*        0.7040        6.6340       -2.2620 
   1   H2P        3.2920        6.9620        3.8010 
   1  H5*1        1.6100        5.1710        6.3250 
   1  H5*2        1.9150        3.4810        5.8640 
   1   H4*        0.2720        3.6790        7.6700 
   1   H3*       -0.4780        2.3500        5.1050 
   1   H2*       -2.5710        3.1300        5.1240 
   1   H1*       -3.0700        4.9300        6.3550 
   1    H8       -3.7640        6.2410        8.2430 
   1    H1       -2.3080        1.5580       12.3880 
   1   H21       -0.9570       -0.0220        9.5680 
   1   H22       -1.4450       -0.2210       11.1800 
   1   H10       -2.8570        1.6910        7.5440 
   1    H9       -0.8070        1.6430        7.8370 
   1    H7        3.4500        7.5240        1.1990 
   1   H61        8.6780        6.0470       -5.4140 
   1   H62        8.0790        4.5280       -4.8120 

