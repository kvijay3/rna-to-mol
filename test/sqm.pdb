ATOM      1  C2   A1     1       5.440   8.216  -4.126  1.00  0.00           C
ATOM      2  N1   A1     1       6.735   7.904  -4.388  1.00  0.00           N
ATOM      3  C6   A1     1       7.348   6.983  -3.567  1.00  0.00           C
ATOM      4  C5   A1     1       6.590   6.404  -2.495  1.00  0.00           C
ATOM      5  C4   A1     1       5.207   6.842  -2.343  1.00  0.00           C
ATOM      6  N3   A1     1       4.608   7.759  -3.157  1.00  0.00           N
ATOM      7  N9   A1     1       4.721   6.153  -1.215  1.00  0.00           N
ATOM      8  C8   A1     1       5.789   5.318  -0.785  1.00  0.00           C
ATOM      9  N7   A1     1       6.906   5.457  -1.507  1.00  0.00           N
ATOM     10  C1*  A1     1       3.353   6.135  -0.770  1.00  0.00           C
ATOM     11  C2*  A1     1       3.202   6.269   0.780  1.00  0.00           C
ATOM     12  C3*  A1     1       2.847   4.837   1.270  1.00  0.00           C
ATOM     13  C4*  A1     1       2.692   3.999  -0.016  1.00  0.00           C
ATOM     14  O4*  A1     1       2.696   4.910  -1.127  1.00  0.00           O
ATOM     15  C5*  A1     1       1.390   3.204  -0.108  1.00  0.00           C
ATOM     16  O5*  A1     1       1.370   2.408  -1.272  1.00  0.00           O
ATOM     17  O3*  A1     1       1.643   4.888   1.994  1.00  0.00           O
ATOM     18  P    G2     2       1.711   4.649   3.593  1.00  0.00           P
ATOM     19  O1P  G2     2       2.913   3.959   4.027  1.00  0.00           O
ATOM     20  O2P  G2     2       1.442   6.154   4.014  1.00  0.00           O
ATOM     21  O5*  G2     2       0.321   3.921   3.884  1.00  0.00           O
ATOM     22  C5*  G2     2       0.311   2.836   4.790  1.00  0.00           C
ATOM     23  C4*  G2     2      -0.608   3.123   5.980  1.00  0.00           C
ATOM     24  C3*  G2     2      -1.980   3.687   5.572  1.00  0.00           C
ATOM     25  C2*  G2     2      -2.122   5.018   6.347  1.00  0.00           C
ATOM     26  C1*  G2     2      -0.880   5.074   7.296  1.00  0.00           C
ATOM     27  O4*  G2     2       0.044   4.085   6.819  1.00  0.00           O
ATOM     28  N9   G2     2      -1.121   4.918   8.708  1.00  0.00           N
ATOM     29  C8   G2     2      -0.623   5.808   9.703  1.00  0.00           C
ATOM     30  N7   G2     2      -0.844   5.383  10.956  1.00  0.00           N
ATOM     31  C5   G2     2      -1.490   4.153  10.811  1.00  0.00           C
ATOM     32  C4   G2     2      -1.663   3.837   9.416  1.00  0.00           C
ATOM     33  N3   G2     2      -2.278   2.697   8.934  1.00  0.00           N
ATOM     34  C2   G2     2      -2.691   1.828   9.892  1.00  0.00           C
ATOM     35  N1   G2     2      -2.575   2.055  11.275  1.00  0.00           N
ATOM     36  C6   G2     2      -1.958   3.221  11.815  1.00  0.00           C
ATOM     37  O6   G2     2      -1.894   3.312  13.048  1.00  0.00           O
ATOM     38  N2   G2     2      -3.224   0.590   9.479  1.00  0.00           N
ATOM     39  O2*  G2     2      -3.360   5.205   6.970  1.00  0.00           O
ATOM     40  O3*  G2     2      -3.040   2.797   5.824  1.00  0.00           O
ATOM     41  O2*  A1     1       2.192   7.197   1.096  1.00  0.00           O
ATOM     42  N6   A1     1       8.656   6.631  -3.758  1.00  0.00           N
ATOM     43  H2   A1     1       4.993   8.963  -4.818  1.00  0.00           H
ATOM     44  H8   A1     1       5.667   4.648   0.074  1.00  0.00           H
ATOM     45  H1*  A1     1       2.783   6.948  -1.313  1.00  0.00           H
ATOM     46  H2*  A1     1       4.125   6.667   1.279  1.00  0.00           H
ATOM     47  H3*  A1     1       3.664   4.413   1.922  1.00  0.00           H
ATOM     48  H4*  A1     1       3.576   3.314  -0.133  1.00  0.00           H
ATOM     49 H5*1  A1     1       0.513   3.902  -0.086  1.00  0.00           H
ATOM     50 H5*2  A1     1       1.327   2.478   0.745  1.00  0.00           H
ATOM     51  H5*  A1     1       1.444   3.005  -2.029  1.00  0.00           H
ATOM     52  H2P  G2     2       1.358   6.796   3.304  1.00  0.00           H
ATOM     53 H5*1  G2     2       1.357   2.626   5.149  1.00  0.00           H
ATOM     54 H5*2  G2     2      -0.107   1.949   4.237  1.00  0.00           H
ATOM     55  H4*  G2     2      -0.719   2.174   6.576  1.00  0.00           H
ATOM     56  H3*  G2     2      -2.031   3.875   4.462  1.00  0.00           H
ATOM     57  H2*  G2     2      -2.034   5.887   5.634  1.00  0.00           H
ATOM     58  H1*  G2     2      -0.338   6.061   7.168  1.00  0.00           H
ATOM     59  H8   G2     2      -0.132   6.748   9.427  1.00  0.00           H
ATOM     60  H1   G2     2      -2.858   1.353  11.924  1.00  0.00           H
ATOM     61  H21  G2     2      -3.439   0.516   8.509  1.00  0.00           H
ATOM     62  H22  G2     2      -3.885   0.165  10.088  1.00  0.00           H
ATOM     63  H10  G2     2      -3.637   4.369   7.369  1.00  0.00           H
ATOM     64  H9   G2     2      -2.930   2.431   6.712  1.00  0.00           H
ATOM     65  H7   A1     1       1.361   6.860   0.729  1.00  0.00           H
ATOM     66  H61  A1     1       9.179   7.009  -4.508  1.00  0.00           H
ATOM     67  H62  A1     1       9.058   5.942  -3.172  1.00  0.00           H
