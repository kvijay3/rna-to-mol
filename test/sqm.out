            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =113

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: P : M.J.S.DEWAR et al. THEOCHEM, 187,1, (1989)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      193
| QMMM:    diag iterations used for timing =        5
| QMMM:
| QMMM:              Internal diag routine = 0.154027 seconds
| QMMM:                 Dspev diag routine = 0.140406 seconds
| QMMM:                Dspevd diag routine = 0.053361 seconds
| QMMM:                Dspevx diag routine = 0.211240 seconds
| QMMM:                 Dsyev diag routine = 0.066912 seconds
| QMMM:                Dsyevd diag routine = 0.075828 seconds
| QMMM:                Dsyevr diag routine = 0.087402 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.030921 seconds
| QMMM:
| QMMM: Using dspevd routine (diag_routine=3).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        6.4190    8.2190   -3.2520
  QMMM:     2        2      N        7.2020    7.5410   -4.1200
  QMMM:     3        3      C        7.1540    6.1900   -4.1140
  QMMM:     4        4      C        6.3210    5.5560   -3.1670
  QMMM:     5        5      C        5.5630    6.3820   -2.3480
  QMMM:     6        6      N        5.5580    7.7270   -2.3360
  QMMM:     7        7      N        4.7890    5.5340   -1.5960
  QMMM:     8        8      C        5.1400    4.2540   -1.9440
  QMMM:     9        9      N        6.0720    4.2280   -2.8760
  QMMM:    10       10      C        3.6560    5.9370   -0.7900
  QMMM:    11       11      C        3.6770    5.6130    0.7270
  QMMM:    12       12      C        2.2840    5.0090    1.0070
  QMMM:    13       13      C        1.4960    5.2890   -0.2740
  QMMM:    14       14      O        2.4780    5.3000   -1.3290
  QMMM:    15       15      C        0.7300    6.6210   -0.3060
  QMMM:    16       16      O        0.0460    6.7580   -1.5520
  QMMM:    17       17      O        1.6060    5.4880    2.1580
  QMMM:    18       18      P        2.1890    5.1370    3.6170
  QMMM:    19       19      O        3.3870    4.2480    3.6380
  QMMM:    20       20      O        2.4740    6.6030    4.1980
  QMMM:    21       21      O        0.9150    4.6250    4.4750
  QMMM:    22       22      C        1.1750    4.2890    5.8400
  QMMM:    23       23      C       -0.0690    3.8560    6.6410
  QMMM:    24       24      C       -0.7800    2.5960    6.1280
  QMMM:    25       25      C       -2.2700    2.9460    6.1630
  QMMM:    26       26      C       -2.3420    4.2980    6.8800
  QMMM:    27       27      O       -1.0460    4.9130    6.7190
  QMMM:    28       28      N       -2.7220    4.3460    8.2820
  QMMM:    29       29      C       -3.4520    5.3830    8.8260
  QMMM:    30       30      N       -3.7180    5.2000   10.1010
  QMMM:    31       31      C       -3.1310    4.0010   10.3980
  QMMM:    32       32      C       -2.5120    3.4470    9.2950
  QMMM:    33       33      N       -1.8550    2.2590    9.2230
  QMMM:    34       34      C       -1.8130    1.6180   10.3680
  QMMM:    35       35      N       -2.3970    2.0780   11.5310
  QMMM:    36       36      C       -3.0880    3.2750   11.6490
  QMMM:    37       37      O       -3.5880    3.6330   12.7090
  QMMM:    38       38      N       -1.1750    0.4280   10.4510
  QMMM:    39       39      O       -3.1400    1.9280    6.6380
  QMMM:    40       40      O       -0.4600    1.4620    6.9400
  QMMM:    41       41      O        3.9250    6.7540    1.5620
  QMMM:    42       42      N        7.9150    5.5000   -5.0430
  QMMM:    43       43      H        6.4910    9.3020   -3.3080
  QMMM:    44       44      H        4.6620    3.3820   -1.5190
  QMMM:    45       45      H        3.5300    7.0160   -0.9360
  QMMM:    46       46      H        4.4540    4.8750    0.9570
  QMMM:    47       47      H        2.3960    3.9200    1.0930
  QMMM:    48       48      H        0.7930    4.4750   -0.4850
  QMMM:    49       49      H        1.4000    7.4810   -0.2200
  QMMM:    50       50      H       -0.0130    6.6780    0.4920
  QMMM:    51       51      H        0.7040    6.6340   -2.2620
  QMMM:    52       52      H        3.2920    6.9620    3.8010
  QMMM:    53       53      H        1.6100    5.1710    6.3250
  QMMM:    54       54      H        1.9150    3.4810    5.8640
  QMMM:    55       55      H        0.2720    3.6790    7.6700
  QMMM:    56       56      H       -0.4780    2.3500    5.1050
  QMMM:    57       57      H       -2.5710    3.1300    5.1240
  QMMM:    58       58      H       -3.0700    4.9300    6.3550
  QMMM:    59       59      H       -3.7640    6.2410    8.2430
  QMMM:    60       60      H       -2.3080    1.5580   12.3880
  QMMM:    61       61      H       -0.9570   -0.0220    9.5680
  QMMM:    62       62      H       -1.4450   -0.2210   11.1800
  QMMM:    63       63      H       -2.8570    1.6910    7.5440
  QMMM:    64       64      H       -0.8070    1.6430    7.8370
  QMMM:    65       65      H        3.4500    7.5240    1.1990
  QMMM:    66       66      H        8.6780    6.0470   -5.4140
  QMMM:    67       67      H        8.0790    4.5280   -4.8120

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -363.5867 kcal/mol        0.9972 kcal/(mol*A)
xmin    20     -367.5350 kcal/mol        1.7233 kcal/(mol*A)
xmin    30     -369.6983 kcal/mol        0.7182 kcal/(mol*A)
xmin    40     -370.3084 kcal/mol        0.7217 kcal/(mol*A)
xmin    50     -370.5564 kcal/mol        0.2187 kcal/(mol*A)
xmin    60     -370.6968 kcal/mol        0.0657 kcal/(mol*A)
xmin    70     -370.7502 kcal/mol        0.1387 kcal/(mol*A)
xmin    80     -370.8004 kcal/mol        0.1725 kcal/(mol*A)
xmin    90     -370.8547 kcal/mol        0.1265 kcal/(mol*A)
xmin   100     -370.9228 kcal/mol        0.1832 kcal/(mol*A)
xmin   110     -370.9446 kcal/mol        0.1321 kcal/(mol*A)
xmin   120     -370.9547 kcal/mol        0.0387 kcal/(mol*A)
xmin   130     -370.9564 kcal/mol        0.0326 kcal/(mol*A)
xmin   140     -370.9583 kcal/mol        0.1075 kcal/(mol*A)
xmin   150     -371.0028 kcal/mol        0.1137 kcal/(mol*A)
xmin   160     -371.0752 kcal/mol        0.0448 kcal/(mol*A)
xmin   170     -371.1163 kcal/mol        0.1363 kcal/(mol*A)
xmin   180     -371.1724 kcal/mol        0.2198 kcal/(mol*A)
xmin   190     -371.2255 kcal/mol        0.0772 kcal/(mol*A)
xmin   200     -371.2825 kcal/mol        0.0645 kcal/(mol*A)
xmin   210     -371.3387 kcal/mol        0.1107 kcal/(mol*A)
xmin   220     -371.4712 kcal/mol        0.2245 kcal/(mol*A)
xmin   230     -371.9452 kcal/mol        0.2308 kcal/(mol*A)
xmin   240     -372.2485 kcal/mol        0.1180 kcal/(mol*A)
xmin   250     -372.6012 kcal/mol        0.1406 kcal/(mol*A)
xmin   260     -372.7874 kcal/mol        0.1698 kcal/(mol*A)
xmin   270     -372.9577 kcal/mol        0.2599 kcal/(mol*A)
xmin   280     -373.0337 kcal/mol        0.0485 kcal/(mol*A)
xmin   290     -373.0440 kcal/mol        0.0710 kcal/(mol*A)
xmin   300     -373.0605 kcal/mol        0.0306 kcal/(mol*A)
xmin   310     -373.0708 kcal/mol        0.0263 kcal/(mol*A)
xmin   320     -373.0798 kcal/mol        0.0292 kcal/(mol*A)
xmin   330     -373.0813 kcal/mol        0.0175 kcal/(mol*A)
xmin   340     -373.0820 kcal/mol        0.0148 kcal/(mol*A)
xmin   350     -373.0836 kcal/mol        0.0078 kcal/(mol*A)
xmin   360     -373.0840 kcal/mol        0.0136 kcal/(mol*A)
xmin   370     -373.0842 kcal/mol        0.0034 kcal/(mol*A)
xmin   380     -373.0842 kcal/mol        0.0014 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -42.6469  -42.4926  -41.5449  -41.4748  -39.9942  -39.4262  -39.2144  -38.9386
 -38.5190  -38.3919  -38.2556  -37.6798  -37.6694  -37.5302  -36.5701  -35.3773
 -34.8793  -34.3702  -34.2480  -33.8206  -32.6031  -32.2320  -31.9040  -30.5427
 -30.2530  -29.5420  -29.2875  -27.0891  -26.8140  -24.9970  -24.7365  -24.4288
 -24.2796  -24.2084  -23.8654  -23.4246  -23.1075  -21.8681  -21.8338  -21.3595
 -20.5354  -20.3733  -19.9975  -19.8734  -19.6176  -19.4357  -19.3311  -19.2530
 -19.0022  -18.6443  -18.5135  -18.2989  -18.1643  -17.7529  -17.5787  -17.5201
 -17.4329  -17.2749  -17.1478  -16.9196  -16.7957  -16.6162  -16.5033  -16.3913
 -16.1496  -16.0457  -15.9936  -15.9149  -15.8469  -15.7024  -15.5702  -15.5174
 -15.2266  -15.1678  -15.0008  -14.9305  -14.8101  -14.7377  -14.6044  -14.5435
 -14.4283  -14.3149  -14.2430  -14.1000  -13.7051  -13.6020  -13.5112  -13.2215
 -13.0617  -13.0100  -12.9562  -12.8122  -12.7229  -12.5630  -12.5288  -12.4813
 -12.3580  -12.1121  -11.8631  -11.8418  -11.7315  -11.6788  -11.5198  -11.5019
 -11.3745  -11.3327  -11.2156  -11.1020  -10.5673  -10.5655  -10.2232   -8.9454
  -8.8590   -0.5762   -0.3550   -0.0635    0.1340    0.4131    0.4541    0.7850
   0.9282    1.1194    1.1680    1.8093    1.8315    1.9935    2.0132    2.0224
   2.1514    2.3078    2.3379    2.3934    2.4207    2.5103    2.5397    2.6138
   2.6684    2.6941    2.8045    2.8115    2.8647    2.8743    2.9936    3.0051
   3.0357    3.0979    3.1578    3.1677    3.2390    3.3152    3.3421    3.3610
   3.4612    3.5181    3.5422    3.5942    3.6290    3.6353    3.6892    3.7621
   3.8604    3.9544    3.9828    4.1151    4.2213    4.2343    4.2893    4.2923
   4.5265    4.5646    4.6466    4.9198    4.9379    4.9457    5.0071    5.0647
   5.2161    5.2210    5.2530    5.4614    5.5347    5.5621    5.6681    5.8177
   5.9508    6.0423    6.1164    6.1301    6.1647    6.2962    6.3388    6.6791
   6.9958

 Heat of formation   =       -373.08419562 kcal/mol  (      -16.17814473 eV)

 Total SCF energy    =    -198979.30759509 kcal/mol  (    -8628.39025173 eV)
 Electronic energy   =   -1732348.47780772 kcal/mol  (   -75120.26702258 eV)
 Core-core repulsion =    1533369.17021263 kcal/mol  (    66491.87677085 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                  0.031
     2      N                 -0.267
     3      C                  0.247
     4      C                 -0.247
     5      C                  0.086
     6      N                 -0.214
     7      N                 -0.208
     8      C                 -0.054
     9      N                 -0.102
    10      C                  0.131
    11      C                 -0.021
    12      C                  0.001
    13      C                 -0.034
    14      O                 -0.277
    15      C                 -0.001
    16      O                 -0.320
    17      O                 -0.744
    18      P                  2.613
    19      O                 -1.071
    20      O                 -0.787
    21      O                 -0.729
    22      C                  0.011
    23      C                 -0.027
    24      C                 -0.014
    25      C                 -0.013
    26      C                  0.127
    27      O                 -0.274
    28      N                 -0.170
    29      C                 -0.076
    30      N                 -0.054
    31      C                 -0.238
    32      C                  0.073
    33      N                 -0.295
    34      C                  0.254
    35      N                 -0.349
    36      C                  0.400
    37      O                 -0.313
    38      N                 -0.341
    39      O                 -0.287
    40      O                 -0.315
    41      O                 -0.310
    42      N                 -0.356
    43      H                  0.191
    44      H                  0.199
    45      H                  0.177
    46      H                  0.153
    47      H                  0.139
    48      H                  0.115
    49      H                  0.088
    50      H                  0.110
    51      H                  0.216
    52      H                  0.300
    53      H                  0.126
    54      H                  0.112
    55      H                  0.123
    56      H                  0.156
    57      H                  0.139
    58      H                  0.141
    59      H                  0.208
    60      H                  0.250
    61      H                  0.228
    62      H                  0.215
    63      H                  0.218
    64      H                  0.217
    65      H                  0.223
    66      H                  0.243
    67      H                  0.251
 Total Mulliken Charge =       0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -2.296   -3.511   -3.388    5.392
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        5.4397    8.2156   -4.1258
  QMMM:     2        2      N        6.7346    7.9041   -4.3881
  QMMM:     3        3      C        7.3481    6.9832   -3.5669
  QMMM:     4        4      C        6.5895    6.4041   -2.4948
  QMMM:     5        5      C        5.2073    6.8420   -2.3430
  QMMM:     6        6      N        4.6080    7.7587   -3.1574
  QMMM:     7        7      N        4.7209    6.1533   -1.2154
  QMMM:     8        8      C        5.7893    5.3180   -0.7853
  QMMM:     9        9      N        6.9063    5.4569   -1.5070
  QMMM:    10       10      C        3.3534    6.1347   -0.7699
  QMMM:    11       11      C        3.2018    6.2693    0.7803
  QMMM:    12       12      C        2.8469    4.8370    1.2704
  QMMM:    13       13      C        2.6916    3.9994   -0.0156
  QMMM:    14       14      O        2.6964    4.9103   -1.1271
  QMMM:    15       15      C        1.3902    3.2038   -0.1081
  QMMM:    16       16      O        1.3703    2.4081   -1.2718
  QMMM:    17       17      O        1.6431    4.8880    1.9939
  QMMM:    18       18      P        1.7111    4.6491    3.5929
  QMMM:    19       19      O        2.9127    3.9595    4.0273
  QMMM:    20       20      O        1.4424    6.1538    4.0139
  QMMM:    21       21      O        0.3207    3.9213    3.8842
  QMMM:    22       22      C        0.3111    2.8364    4.7903
  QMMM:    23       23      C       -0.6077    3.1231    5.9796
  QMMM:    24       24      C       -1.9803    3.6873    5.5716
  QMMM:    25       25      C       -2.1224    5.0175    6.3467
  QMMM:    26       26      C       -0.8805    5.0741    7.2962
  QMMM:    27       27      O        0.0443    4.0854    6.8185
  QMMM:    28       28      N       -1.1211    4.9177    8.7078
  QMMM:    29       29      C       -0.6234    5.8078    9.7025
  QMMM:    30       30      N       -0.8437    5.3832   10.9560
  QMMM:    31       31      C       -1.4903    4.1530   10.8108
  QMMM:    32       32      C       -1.6635    3.8373    9.4158
  QMMM:    33       33      N       -2.2777    2.6966    8.9341
  QMMM:    34       34      C       -2.6911    1.8277    9.8922
  QMMM:    35       35      N       -2.5748    2.0554   11.2750
  QMMM:    36       36      C       -1.9576    3.2209   11.8147
  QMMM:    37       37      O       -1.8941    3.3119   13.0476
  QMMM:    38       38      N       -3.2237    0.5904    9.4786
  QMMM:    39       39      O       -3.3596    5.2047    6.9700
  QMMM:    40       40      O       -3.0398    2.7970    5.8243
  QMMM:    41       41      O        2.1915    7.1966    1.0963
  QMMM:    42       42      N        8.6557    6.6308   -3.7577
  QMMM:    43       43      H        4.9933    8.9630   -4.8182
  QMMM:    44       44      H        5.6666    4.6477    0.0740
  QMMM:    45       45      H        2.7829    6.9479   -1.3129
  QMMM:    46       46      H        4.1254    6.6673    1.2790
  QMMM:    47       47      H        3.6639    4.4129    1.9219
  QMMM:    48       48      H        3.5761    3.3139   -0.1332
  QMMM:    49       49      H        0.5129    3.9023   -0.0862
  QMMM:    50       50      H        1.3265    2.4782    0.7448
  QMMM:    51       51      H        1.4443    3.0046   -2.0286
  QMMM:    52       52      H        1.3578    6.7956    3.3042
  QMMM:    53       53      H        1.3574    2.6261    5.1493
  QMMM:    54       54      H       -0.1072    1.9486    4.2365
  QMMM:    55       55      H       -0.7186    2.1738    6.5757
  QMMM:    56       56      H       -2.0311    3.8750    4.4622
  QMMM:    57       57      H       -2.0345    5.8874    5.6343
  QMMM:    58       58      H       -0.3376    6.0606    7.1678
  QMMM:    59       59      H       -0.1320    6.7479    9.4268
  QMMM:    60       60      H       -2.8580    1.3534   11.9243
  QMMM:    61       61      H       -3.4392    0.5163    8.5089
  QMMM:    62       62      H       -3.8847    0.1649   10.0877
  QMMM:    63       63      H       -3.6373    4.3688    7.3688
  QMMM:    64       64      H       -2.9303    2.4314    6.7118
  QMMM:    65       65      H        1.3606    6.8600    0.7288
  QMMM:    66       66      H        9.1786    7.0088   -4.5076
  QMMM:    67       67      H        9.0577    5.9425   -3.1719

           --------- Calculation Completed ----------

